modules/openapi-generator/src/main/resources/python/api_test.mustache

```diff
@@ -27,8 +39,14 @@ class {{#operations}}Test{{classname}}(unittest.TestCase):
 {{#summary}}
         {{{summary}}}  # noqa: E501
 {{/summary}}
+
+{{#allParams}}
+        :param {{dataType}} {{paramName}}:{{#description}} {{{description}}}{{/description}}{{#required}} (required){{/required}}{{#optional}}(optional{{#defaultValue}}, default to {{{.}}}{{/defaultValue}}){{/optional}}
+{{/allParams}}
+
```

Comment:
These api_test and model_test updates will impact all of the tests that are auto generated when the users run the generator.
This pull request does not yet include any tests of this new functionality.
Tests will need to be added showing that both the new model tests and the api client tests  produce code which runs and passes tests.
If these tests were omitted this update could breaking the tests produced in the python client. If this is too much work for this PR the model_test update and the api_test update can be broken up in to different PRs.

Reply:
I removed all `samples/client/petstore/python/test/*`, and did a `./bin/python-petstore-all.sh`. In general, the code produced by the python-experimental and the "normal"python generators are quite different. I managed most of tests for the models only. The only serious problem I found are described in the answer to the previous comment (https://github.com/OpenAPITools/openapi-generator/pull/4514#discussion_r349696492).

Reply:
Can you resolve this conversation?